packages/paste-website/src/pages/form-elements/select/index.mdx

```diff
@@ -0,0 +1,944 @@
+---
+title: Select - Form Elements
+package: '@twilio-paste/form'
+description: A select allows users to select an option from a list in a form field.
+---
+
+import {graphql} from 'gatsby';
+import {FormLabel, FormHelpText, FormInput, Select, Option, OptionGroup, CheckboxGroup, Checkbox} from '@twilio-paste/form';
+
+import {Anchor} from '@twilio-paste/anchor';
+import {Box} from '@twilio-paste/box';
+import {Button} from '@twilio-paste/button';
+import {InformationIcon} from '@twilio-paste/icons/esm/InformationIcon';
+import {UnorderedList, ListItem} from '@twilio-paste/list';
+import {Callout, CalloutTitle, CalloutText} from '../../../components/callout';
+import {DoDont, Do, Dont} from '../../../components/DoDont';
+import {Codeblock} from '../../../components/codeblock';
+import {SidebarCategoryRoutes} from '../../../constants';
+import Changelog from '@twilio-paste/form/CHANGELOG.md';
+
+export const pageQuery = graphql`
+  {
+    allPasteComponent(filter: {name: {eq: "@twilio-paste/form"}}) {
+      edges {
+        node {
+          name
+          description
+          status
+          version
+        }
+      }
+    }
+    mdx(fields: {slug: {eq: "/form-elements/select/"}}) {
+      headings {
+        depth
+        value
+      }
+    }
+  }
+`;
+
+<ComponentHeader
+  name="Select"
+  categoryRoute={SidebarCategoryRoutes.FORM}
+  githubUrl="https://github.com/twilio-labs/paste/tree/master/packages/paste-core/components/form"
+  storybookUrl="https://twilio-labs.github.io/paste/?path=/story/components-form--select-options"
```

Comment:
Clicking this link takes me to Alerts. I wonder if this is a case of production storybook?

Reply:
Yeah me too, that's odd 🤔 I'll look into it after I address the other PR feedback comments.